wiki/Main_Page/tr.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: main_page
+---
+
+
+<div class="wiki-main-page__blurb">
+Osu! wiki'sine hoşgeldin, osu! ile ilgili çok çeşitli bilgiler içeren bir proje.
+</div>
+
+
+<div class="wiki-main-page__panels">
+<div class="wiki-main-page-panel">
+
+
+# Başlarken
+
+[Hoşgeldin](/wiki/Welcome) • [Kayıt](/wiki/Registration) • [Kurulum](/wiki/Installation) • [Oyun Arayüzü](/wiki/Interface) • [Oyun Ayarları](/wiki/Options) • [Kısayol Tuşları](/wiki/Shortcut_key_reference) • [Site Haritası](/wiki/Sitemap) • [Kılavuzlar](/wiki/Guides) • [Yardım Merkezi](/wiki/Help_Centre)
+
+</div>
+<div class="wiki-main-page-panel">
+
+
+# Oynanış
+
+[SSS](/wiki/FAQ) • [Çoklu](/wiki/Multi) • [Oyun Modları](/wiki/Game_Modes): ([osu!](/wiki/Game_Modes/osu!) • [osu!taiko](/wiki/Game_Modes/osu!taiko) • [osu!catch](/wiki/Game_Modes/osu!catch) • [osu!mania](/wiki/Game_Modes/osu!mania))
+
+</div>
+<div class="wiki-main-page-panel">
+
+
+# [Beatmap Editörü](/wiki/Beatmap_Editor)
+
+Bölümler: ([Oluşturma](/wiki/Beatmap_Editor/Compose) • [Dizayn](/wiki/Beatmap_Editor/Design) • [Zamanlama](/wiki/Beatmap_Editor/Timing) • [Şarkı Kurulumu](/wiki/Beatmap_Editor/Song_Setup)) • Bileşenler: ([Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance Snap](/wiki/Beatmap_Editor/Distance_Snap)) • [Beatmapping](/wiki/Beatmapping) • [Difficulties](/wiki/Difficulties) • [Vuruş Objeleri](/wiki/Hit_Objects) • [Mapping Teknikleri](/wiki/Mapping_Techniques) • [Film Şeritleme](/wiki/Storyboarding)
+
+</div>
+<div class="wiki-main-page-panel">
+
+
+# Çevrimiçi Düzenleme ve Sıralama
+
+[Sunuş](/wiki/Submission) • [Beatmap Onayı](/wiki/Beatmap_ranking_procedure) • [Modlama](/wiki/Modding) • [Sıralama Kriterleri](/wiki/Ranking_Criteria): ([osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) • [osu!mania](/wiki/Ranking_Criteria/osu!mania))
+
+</div>
+<div class="wiki-main-page-panel">
+
+
+# Osu! Dünyası
+
+[Sözlük](/wiki/Glossary) • [Turnuvalar](/wiki/Tournaments) • [Stilleme](/wiki/Skinning) • [Projeler](/wiki/Projects)
```

Comment:
"Skinning" sözcüğü oyunun pek çok yerinde "Tema" diye çevirildiği için burada da buna sadık kalınması gerekir
```suggestion
[Sözlük](/wiki/Glossary) • [Turnuvalar](/wiki/Tournaments) • [Temalar](/wiki/Skinning) • [Projeler](/wiki/Projects)
```

Reply:
@zentoryny  Değiştirilmediği halde *resolved* olarak işaretlemişsin. "Stilleme" kelimesini kimse "Skinning" için kullanmıyor, türkçe bir kelime kullanılacaksa da "Tema Yaratma" şeklinde kullanılıyor. Bu değişikliğin tekrar gözden geçirilmesi gerektiğini düşünüyorum. Consistency is key!